microsetta_private_api/db/patches/0140.sql

```diff
@@ -0,0 +1,56 @@
+-- May 13, 2024
+-- Create table to store observation categories
+CREATE TABLE sample_observation_categories (
```

Comment:
```suggestion
CREATE TABLE barcodes.sample_observation_categories (
```

Apologies for missing this the first time around, but let's make sure to specify a schema when creating tables (this applies to this and all following queries, don't want to needlessly replicate the comment). Ideally, we want all new queries (whether in DB patches or repos) to specify the schema as well. Some legacy code doesn't, and queries work without it, but specificity helps prevent issues down the road.